### DISCUSSION TITLE
Incorrect instructions on finding tracker.

### DISCUSSION THREAD

**Discussion by gibsop1** [seemoo-lab/AirGuard#109] [Q&A]
There is a typo in the instructions on what to do if you're being tracked. It says "do not move to a safe location before deactivating" I presume this should say "move to a safe location"

**Comment by DominikNovosel** [comment]
No, I'm pretty sure it's correct. Assuming "safe location" means something like your home, school, workplace or other location where you may regularly reside, it would be illogical and dangerous to go to any of these before deactivating the tracker and making sure no one can follow you.

**Comment by lbdroid** [comment]
Yes -- don't lead baddies to your home.
However, the local police station could also be considered a safe place, and leading baddies there would be an excellent idea!
